games-action/minetest/metadata.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<herd>games</herd>
```

Comment:
That was a false statement from @mgorny. Except for him, there is no consensus on agreeing to ignore them. They are still reviewing ebuilds, see https://github.com/gentoo/gentoo/commit/b964bb05fe54c61122e225a623d6349f11c7363b

A clean solution to this involves a ComRel action or even the council. None of this has to be discussed in this PR.